extensions/python/package.json

```diff
@@ -10,8 +10,7 @@
   },
   "extensionKind": [
     "ui",
-    "workspace",
-    "web"
+    "workspace"
```

Comment:
Should we even have this field? VS Code removed it a while back it seems : https://github.com/microsoft/vscode/commit/03ca148fd7e68c8fa86c94bda4be6273c73e2b0d

Reply:
Thanks for pointing it out. Should I remove this or wait for the merge to remove it?